legal-api/src/legal_api/reports/report.py

```diff
@@ -440,12 +440,12 @@ def _format_address(address):
         return address
 
     @staticmethod
-    def _populate_business_info_to_filing(filing: Filing, legal_entity: LegalEntity):
-        founding_datetime = LegislationDatetime.as_legislation_timezone(legal_entity.founding_date)
+    def _populate_business_info_to_filing(filing: Filing, business: any):
+        founding_datetime = LegislationDatetime.as_legislation_timezone(business.founding_date)
```

Comment:
I don't think AlternateName entities have a `founding_date` attribute. Maybe we do [something like this](https://github.com/bcgov/lear/pull/2522/files#diff-21be2c43c7121075d95bf3a989eff70fec5a03fa599fe0f5217d2c565cd8d6a1R359) and use `start_date` for alternate name entities?